templates/layouts/main.html

```diff
@@ -45,7 +45,7 @@
 
   <header class="site-header">
     <div class="disclaimer">
-      <span class="disclaimer__left">Looking for <a href="http://www.fec.gov">fec.gov?</a></span>
+      <span class="disclaimer__left">Looking for <a href="http://www.fec.gov">FEC.gov?</a></span>
```

Comment:
I'll merge this for now to keep things consistent, and then if we want to make a universal change we can.